app/src/bcsc-theme/features/onboarding/components/PrivacyPolicyContent.tsx

```diff
@@ -46,7 +49,7 @@ export const PrivacyPolicyContent: React.FC<PrivacyPolicyContentProps> = ({
 
   const handlePressLearnMore = async () => {
     try {
-      await Linking.openURL(SECURE_APP_LEARN_MORE_URL)
+      navigation.navigate(BCSCScreens.OnboardingSecureLearnMore)
```

Comment:
context: These 'content' components are used to prevent coupling the screen content with the stack specific navigation.

This means that any navigation controls will need to be passed in as a prop (ie: onPressLearnMore)

see: OnbardingPrivacyPolicyScreen.tsx